src/Exception/CommandInvokerException.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Dhii\Invocation\Exception;
+
+use Exception as RootException;
+use Dhii\Invocation\CommandInvokerInterface;
+
+/**
+ * An exception that occurs in relation to a command invoker.
+ *
+ * @since [*next-version*]
+ */
+class CommandInvokerException extends AbstractBaseCommandInvokerException
+{
+    /**
```

Comment:
Consider using `NormalizeStringCapableTrait` and `NormalizeIntCapableTrait` here instead of in `AbstractBaseCommandInvokerException`.